test/e2e/test_image.py

```diff
@@ -125,7 +125,7 @@ def test_create_with_file_extension_content_type_mismatch(self):
         self.post_image_with_file_extension_content_type_mismatch(
             IMAGE_POST_METADATA_DATA_REQUIRED_VALUES_ONLY, "image.jpg"
         )
-        self.check_post_image_failed_with_detail(422, "Filename does not contain the correct extension")
+        self.check_post_image_failed_with_detail(422, "File extension and content type do not match")
```

Comment:
I think there should be another e2e test for a file type that is known not to be allowed, rather than just mismatching.